pkg/component/operator/video/v0/video_operation.go

```diff
@@ -186,19 +192,47 @@ func subsampleVideoFrames(input *structpb.Struct) (*structpb.Struct, error) {
 	return base.ConvertToStructpb(output)
 }
 
-func getFramesKwArgs(inputStruct SubsampleVideoFramesInput) ffmpeg.KwArgs {
-	kwArgs := ffmpeg.KwArgs{"vf": "fps=" + fmt.Sprintf("%d", inputStruct.Fps)}
+func getFramesKwArgs(inputStruct SubsampleVideoFramesInput) (ffmpeg.KwArgs, error) {
```

Comment:
Why do we need to return error here?
It seems there is no case to return error

Reply:
left that in case a string processing function is implemented. will remove